### DISCUSSION TITLE
除了文档的两种引入方式，可以手动 import 引入么，使用那个引入那个

### DISCUSSION THREAD

**Discussion by warmthsea** [Moonofweisheng/wot-design-uni#132] [General]
https://wot-design-uni.cn/guide/quick-use.html

**Comment by Moonofweisheng** [comment]
> https://wot-design-uni.cn/guide/quick-use.html

可以，不过其实按文档提供的方式引入，也不会打包没用到的组件的。

**Reply by warmthsea** [reply]
文档似乎没有提供这样的写法

/.vue
```
   <script setup lang="ts">
       import { WdButton } from ‘wot-design-uni’
   </script>
   
   <template>
        <wd-button>主要按钮</wd-button>
   </template>
```

**Comment by Moonofweisheng** [comment]
> 文档似乎没有提供这样的写法
> 
> /.vue
> 
> ```
>    <script setup lang="ts">
>        import { WdButton } from ‘wot-design-uni’
>    </script>
>    
>    <template>
>         <wd-button>主要按钮</wd-button>
>    </template>
> ```
因为组件库并未将所有组件在入口文件中导出，所以如果需要导入的话需要指定到具体组件的文件路径。

**Reply by warmthsea** [reply]
明白了，谢谢作者回复
